contrib/kubespray/roles/requirement/devbox/tasks/ubuntu.yml

```diff
@@ -37,5 +37,39 @@
   when:
     - dev_box_docker_version_replace is version('1.10.0', '<=')
 
+- name: "Dev-box 3.1 Check whether the vm can raw.githubusercontent.com"
+  raw: ping raw.githubusercontent.com -c 3
```

Comment:
Can we use a `wget` to download some file from this site? like `wget https://raw.githubusercontent.com/microsoft/pai/master/README.md`. Because sometimes we can ping it, but cannot access it.

Reply:
change to curl